hudi-integ-test/prepare_integration_suite.sh

```diff
@@ -167,5 +167,10 @@ $move_to_root && $final_command
 # change back to original working directory
 cd $_CALLING_DIR
 
+printf "Docker stats start XXX "
+docker stats -a --no-stream
```

Comment:
Looks like this is not getting printed
https://dev.azure.com/apache-hudi-ci-org/apache-hudi-ci/_build/results?buildId=4274&view=logs&j=3b6e910d-b98f-5de6-b9cb-1e5ff571f5de&t=30b5aae4-0ea0-5566-42d0-febf71a7061a